.travis.yml

```diff
@@ -16,7 +16,6 @@ env:
     - DOCKER_VERSION=17.09.0
     - DOCKER_COMPOSE_VERSION=1.17.1
   matrix:
-    - TARGET="71"
```

Comment:
Would this all work but keep this target + the Docker file in? Or will all the tests fail because of the composer requirement?

Reply:
I think this won't work because of the composer requirement, at least if `--ignore-platform-reqs` is not used